cypress/platform/xss4.html

```diff
@@ -88,7 +88,7 @@
       let diagram = '%%{init: {"flowchart": {"htmlLabels": "true"}} }%%\n';
       diagram += 'flowchart\n';
       diagram += 'A["<ifra';
-      diagram += "me srcdoc='<scrip";
+      diagram += "me srcdoc='<script";
```

Comment:
Seems this is wrong because `t` letter was deliberately placed on the following line

Reply:
thanks for catching, added handling of 'pragma: codespell-ignore' comments to ignore such lines.